tools/walletextension/storage/sqlite.go

```diff
@@ -0,0 +1,113 @@
+package storage
+
+import (
+	"database/sql"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/crypto/ecies"
+	_ "github.com/mattn/go-sqlite3" // sqlite driver for sql.Open()
+	"github.com/obscuronet/go-obscuro/go/rpc"
+)
+
+type SqliteDatabase struct {
+	db       *sql.DB
+	hostAddr string
```

Comment:
more generally, I don't quite follow the purpose of the "hostAddr".
Let's remove it

Reply:
removed  (see the comment above)